logprep/util/schema_and_rule_checker.py

```diff
@@ -138,14 +139,17 @@ def _validate_rules_in_path(
         path_schema: str = None,
     ):
         number_of_checked_rules = 0
-        for root, _, files in walk(path_rules):
-            for file in files:
-                number_of_checked_rules += 1
-                rule_path = join(root, file)
-
-                multi_rule = self.check_rule_creation_errors(rule_class, rule_path)
-                self._validate_schema(multi_rule, path_schema, rule_path)
-            self._print_schema_check_results(path_schema)
+        if isinstance(path_rules, dict):
```

Comment:
Same problem here with the `path_rules`, though this might be indeed actually best solved by just renaming the variable.